app/views/user_mailer/sharing_notification.html.erb

```diff
@@ -3,7 +3,10 @@
   user_email = @user.email
   link = url_for(action: 'show', controller: 'plans', id: @role.plan.id)
   helpdesk_email = Rails.configuration.branding[:organisation][:helpdesk_email]
-  contact_us_url = Rails.configuration.branding[:organisation][:contact_us_url]
+
+  contact_us = Rails.configuration.branding[:organisation][:contact_us_url] ||= contact_us_url

```

Comment:
please, avoid the usage of memoized operator here. A ternary operator would fit nicer.